app/views/organisations/_videos.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="brand--<%= @organisation.brand %> brand__border-color organisation__brand-border-top organisation__section-wrap organisation__margin-bottom" id="videos">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/heading", {
+          text: t("organisations.videos"),
+          padding: true,
+          lang: "en"
+      } %>
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render "govuk_publishing_components/components/govspeak" do %>
+        <a href="<%= @documents.promotional_video_feature_youtube_url %>" class="govuk-link brand__color"><%= @documents.promotional_video_feature_youtube_url %></a>
+      <% end %>
+      <p class="govuk-heading-s govuk-!-margin-top-1"><%= @documents.promotional_video_feature_title %></p>
+    </div>
+    <div class="govuk-grid-column-one-third govuk-!-margin-top-1">
+     <%= render "govuk_publishing_components/components/list",
+      extra_spacing: true,
+      items: @documents.promotional_video_feature_item_links(@organisation).map { |item| sanitize("<a href=\"#{item[:href]}\" rel=\"noopener\" class=\"govuk-link brand__color\">#{item[:text]}</a>") }
+      %>
+      <p class="govuk-body govuk-!-margin-top-7"><%= link_to(t("organisations.more_videos_link"), @organisation.more_videos_link, { class: "govuk-link brand__color" }) %></p>
```

Comment:
I think the current requirement is to remove this link. I'm reviewing this commit by commit, so it's possible it's addressed in a future commit, if so can you merge that change with this commit?

Reply:
@andysellick I can't find this requirement - I remember in the last meeting I was confused about a requirement that was added to remove "Summary text" called `Latest videos from the 10 Downing Street YouTube channel` - but I had never put that in the UI at all. In that same meeting I was asked to add spacing to this link, so I'm not sure this link would need to be removed?

But yep, Keith has been squashing commits 👍